light_curves/code/sample_selection.py

```diff
@@ -360,7 +361,7 @@ def get_SDSS_sample(coords, labels, num, verbose=1):
     """
     # Define the query
     query = "SELECT TOP " + str(num) + " specObjID, ra, dec, z FROM SpecObj \
-    WHERE ( z > 0.0 AND z < 2.0 AND class='QSO' AND zWARNING=0 )"
+    WHERE ( z > 0.0 AND z < 1.0 AND class='QSO' AND zWARNING=0 )"
```

Comment:
let's make the max_redshift a variable of get_SDSS_sample().  Sorry I didn't think of this earlier.